Adjust dependencies

The adjustment to libgpg-error-sys is related to changes in how gpgme
configuration works.

The adjustment to winreg isn't particularly useful since we aren't
shipping anything useful for building windows, but it makes the
installation on debian easier.

Forwarded: not needed
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -30,8 +30,8 @@
 version = "0.2"
 
 [dependencies.libgpg-error-sys]
-version = "0.5.1"
+version = "0.5.2"
 [target."cfg(windows)".build-dependencies.winreg]
-version = "0.9"
+version = "0.6"
 [badges.maintenance]
 status = "experimental"
